packages/synapse-react-client/src/components/ChallengeTeamWizard/ChallengeTeamWizard.tsx

```diff
@@ -478,14 +363,25 @@ const ChallengeTeamWizard: React.FunctionComponent<
         return (
           <RegistrationSuccessful
             createdNewTeam={createdNewTeam}
-            teamName={selectedTeam?.name}
+            teamId={selectedTeamId}
           />
         )
       case StepsEnum.CREATE_NEW_TEAM:
         return (
           <CreateChallengeTeam
-            onChangeTeamInfo={handleChangeTeamInfo}
-            onError={setErrorMessage}
+            ref={createChallengeTeamRef}
+            challengeId={challenge!.id}
+            onCanSubmitChange={canCreateTeam => {
+              setStep(step => ({
+                ...step,
+                confirmEnabled: canCreateTeam,
+              }))
+            }}
```

Comment:
Toggle the `disabled` attribute of the confirm button based on the form validity (i.e. the team must have a name and there must be <=3 emails to invite to the team)